zyngine/zynthian_controller.py

```diff
@@ -137,13 +137,15 @@ def _configure(self):
 					self.value_min = 0
 				if self.value_max == None:
 					self.value_max = n - 1
-				value_range = self.value_max - self.value_min + 1
-				if self.is_integer:
+				value_range = self.value_max - self.value_min
+				if n == 1:
+					self.ticks.append(self.value_min)
+				elif self.is_integer:
 					for i in range(n):
-						self.ticks.append(self.value_min + int(i * value_range / n))
+						self.ticks.append(self.value_min + int(i * value_range / (n - 1)))
 				else:
 					for i in range(n):
-						self.ticks.append(self.value_min + i * value_range / n)
```

Comment:
This should be done in a separate PR. It affects all engines!

Reply:
Possibly but this line of code is not used by any engine until the new pianoteq implementation invokes it. I loaded every type of engine in testing branch and none trigger this line.

Reply:
Actually, the change before this one is more impactful and needs more consideration.

Reply:
I have submitted https://github.com/zynthian/zynthian-ui/pull/459

Reply:
I have created individual PRs for each change that has wider impact. They are also consolidated into this PR.